docs/TriageProcess.md

```diff
@@ -0,0 +1,61 @@
+In many situations, these issues turn out to be a result of some kind of misconfiguration in the user code.
+In some rare situations, however, these turn out to be caused by very impactful issues. So we will make a call during the triage whether we need to immediately investigate certain issues or not.
+If not, we will move the investigation to the `Next Sprint Planning` to review during the [next sprint planning meeting](#milestone-planning).
+
+### Documentation requests
+Some issues turn out to indicate user confusion around how to configure different aspects of the framework.
+When we determine such isssues, we will mark these with the `Docs` label and move them into the `Next Sprint Planning` milestone to handle at a later time. The goal here will be to fill in the gaps or clarify our documentation, so that customers can be successful by using the guidance provided in the documentation.
+If we identify a documentation issue which too many customers are having trouble with, we may choose to address that in current milestone.
+
+## Milestone Planning
+Our milestones are usually a month long.
+Before each milestone we have one or more planning meetings, where we look through all the accumulated issues in the `Next Sprint Planning` milestone and choose the most important and impactful ones to handle during the next milestone. This will be a mixture of feature requests, bug fixes, documentation issues as well as some investigations.
+
+Note, that we will investigate only issues, which have accumulated more than certain number of upvotes and/or comments, which will indicate that there is some wider impact associated with it.
+"We may not investigate issues which haven't received many votes/comments and choose to close these. The reason is that the impact is very scoped and potentially something is wrong in the user code.
+
+For some feature requests and bug reports, depending on the user involvement, we may choose to move these to the backlog at this point. What this means, is that they will not be looked at again up until the next major release planning.
+
+## Release Planning
+Once we approach to the end of the release cylce (.NET Core 3, .NET 5) we will look through the accumulated issues in the `Backlog` milestone. This is a long process as the amount of issues accumulated in this milestone is quite large.
+
+We will try to prioritize issues with most community requests / upvotes assuming these are aligned with our goals.
+Issues, which we will think are candidates for the upcoming release, will be moved to the `Next Sprint Planning` milestone.
+
+## Process Visualization
+If you're a type of person who likes to see things visualized, the following diagram summarizes the processes detailed above:
```

Comment:
```suggestion
The following diagram summarizes the processes detailed above:
```

Reply:
The chart is missing an arrow from Release Planning -> Next Sprint Planning, right now it shows everything getting closed 😁.